com.unity.render-pipelines.universal/Runtime/ScriptableRenderer.cs

```diff
@@ -499,6 +499,8 @@ internal struct RTHandleRenderTargetIdentifierCompat
         RTHandleRenderTargetIdentifierCompat m_CameraDepthTarget;
         RTHandleRenderTargetIdentifierCompat m_CameraResolveTarget;
 
+        HDRFormat m_HDRFormat;
```

Comment:
I wonder if you could store and initialize this as part of CameraData instead.

Reply:
I guess you actually already have this in `cameraData.cameraTargetDescriptor.graphicsFormat` as well which you initialize here? https://github.com/Unity-Technologies/Graphics/pull/6728/files?file-filters%5B%5D=.cs&show-viewed-files=true#diff-0eb17165c501725142edafebc19d4d08538deb40c78a8cb059741641db83581cR1287

Reply:
done.